tests/testthat/test-export-dataset.R

```diff
@@ -158,4 +195,21 @@ with_test_authentication({
         expect_identical(names(df2), c("v1", "v2", "v4"))
         expect_identical(is.na(df2$v1), is.na(df$v1))
     })
+
+    test_that("We can export hidden variables", {
+        skip_locally("Vagrant host doesn't serve files correctly")
+        ds$hidden_var <- 1:20
+        ds <- hideVariables(ds, "hidden_var")
+        filename <- tempfile()
+
```

Comment:
We should also have a test in this pattern that `write.csv(ds, file = filename)` (ie `include.hidden = FALSE`) returns no hidden variables.